### PR TITLE
chore(bridge-ui-v2): Improve tooltip ux

### PR DESCRIPTION
**Pull request by mintcnn**
- update dotenv config to fix error 'env not found' when running `pnpm dev` on Node 18
- improve the UX of the tooltip
    - open the tooltip when hover or click
    - close the tooltip when the mouse leaves or click outside with some delay

### CODE REVIEW

**Comment by KorbinianK** [comment]
There is nothing wrong with the env file loading. It's expected in the package and works just fine.

**Comment by mintcnn** [comment]
> There is nothing wrong with the env file loading. It's expected in the package and works just fine.
Reproduced! but did not find the error again. I will revert the change for dotenv!

File: packages/bridge-ui-v2/src/components/Tooltip/Tooltip.svelte [hunk 1]
```diff
@@ -31,22 +33,21 @@
...
   });
 
   onDestroy(() => {
     closeTooltip();
-    document.removeEventListener('click', closeTooltip);
   });
 </script>
 
-<div class={classes}>
+<div class={classes} role="presentation" on:mouseleave={() => closeTooltip(1000)}>
```
Comment (KorbinianK) [inline packages/bridge-ui-v2/src/components/Tooltip/Tooltip.svelte:43]:
```suggestion
<div class={classes} role="presentation" on:mouseleave={() => closeTooltip(200)}>
```

**Review by KorbinianK — DISMISSED** [review]

**Comment by KorbinianK** [comment]
@mintcnn can you rebase your branch to `alpha-6` please? Then we merge it. I changed the timeout to 200ms, but I like the behaviour.

**Comment by mintcnn** [comment]
> Contributor
@KorbinianK 
I already created the new PR since rebasing to alpha-6 has a lot of changes and conflict files, please help review again
https://github.com/taikoxyz/taiko-mono/pull/15449

**Pull request closed without merging**